test/test_eos_unit.cpp

```diff
@@ -232,7 +232,7 @@ SCENARIO("EOS Variant Type", "[Variant][EOS]") {
   std::cout << demangle(typeid(EOS).name()) << std::endl;
 }
 
-SCENARIO("EOS Builder and Modifiers", "[EOSBuilder],[Modifiers][IdealGas]") {
+SCENARIO("EOS Builder and Modifiers", "[EOSBuilder][Modifiers][IdealGas]") {
```

Comment:
This was a harmless bug. The comma never should have been there. We need it not there, in fact, to correctly parse this string for categories with catch.